src/classes/i18n.js

```diff
@@ -217,86 +196,72 @@ export default class i18n {
 
         const translation_file_path = this.config.get('locale_dir') + lang + '.json';
 
-        new Promise(async (resolve, reject) => {
-            if (
-                process.env.REFRESH_TRANSLATIONS
-                || !await this.version_match(translation_file_path)
-            ) {
-                // force re-download translations
-                await this.download_translations(url, translation_file_path);
-            }
-
+        let retry_count = 0;
 
-            let retry_count = 0;
+        async.doWhilst(
```

Comment:
V3 returns a promise if no callback is passed
https://caolan.github.io/async/v3/docs.html#doWhilst

> Returns:
> a promise, if no callback is passed

Reply:
Fixed: https://github.com/anyTV/anytv-i18n/pull/10/commits/947192cbee484268b862652afa2ebf3c16065ed7